### ISSUE TITLE
node dependencies need updating

### ISSUE THREAD

**Issue by gbolo** [gbolo/aws-power-toggle#53]
`npm install` produces the following alarming message:
```
found 2350 vulnerabilities (1494 low, 33 moderate, 821 high, 2 critical)
```

Also the build for `make docker` fails:

```
▶ building web frontend ui...

> yorkie@2.0.0 install /opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/yorkie
> node bin/install.js

setting up Git hooks
can't find .git directory, skipping Git hooks installation

> node-sass@4.10.0 install /opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/node-sass
> node scripts/install.js

Downloading binary from https://github.com/sass/node-sass/releases/download/v4.10.0/linux_musl-x64-64_binding.node
Download complete
Binary saved to /opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/node-sass/vendor/linux_musl-x64-64/binding.node
Caching binary to /root/.npm/node-sass/4.10.0/linux_musl-x64-64_binding.node

> node-sass@4.10.0 postinstall /opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/node-sass
> node scripts/build.js

Binary found at /opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/node-sass/vendor/linux_musl-x64-64/binding.node
Testing binary
Binary is fine
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 1565 packages from 893 contributors and audited 1634 packages in 24.784s
found 2350 vulnerabilities (1494 low, 33 moderate, 821 high, 2 critical)
  run `npm audit fix` to fix them, or `npm audit` for details

> frontend@0.1.0 build /opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend
> vue-cli-service build


Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
 ERROR  RangeError: Unsupported time zone specified undefined
RangeError: Unsupported time zone specified undefined
    at new DateTimeFormat (native)
    at Date.toLocaleTimeString (native)
    at Debugger.title (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/friendly-errors-webpack-plugin/src/output.js:54:42)
    at FriendlyErrorsWebpackPlugin.displayErrors (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/friendly-errors-webpack-plugin/src/friendly-errors-plugin.js:104:12)
    at doneFn (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/friendly-errors-webpack-plugin/src/friendly-errors-plugin.js:56:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:18:1)
    at AsyncSeriesHook.lazyCompileHook (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/tapable/lib/Hook.js:154:20)
    at emitRecords.err (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/webpack/lib/Compiler.js:257:22)
    at Compiler.emitRecords (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/webpack/lib/Compiler.js:371:39)
    at emitAssets.err (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/webpack/lib/Compiler.js:251:10)
    at hooks.afterEmit.callAsync.err (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/webpack/lib/Compiler.js:357:14)
    at _err0 (eval at create (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:20:1)
    at callback (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/copy-webpack-plugin/dist/index.js:126:17)
    at afterEmit (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/copy-webpack-plugin/dist/index.js:220:13)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/tapable/lib/HookCodeFactory.js:32:10), <anonymous>:7:1)
    at AsyncSeriesHook.lazyCompileHook (/opt/gopath/src/github.com/gbolo/aws-power-toggle/frontend/node_modules/tapable/lib/Hook.js:154:20)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend@0.1.0 build: `vue-cli-service build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-11T02_41_39_063Z-debug.log
make: *** [Makefile:51: frontend] Error 1
+ cp -rp testdata/sampleconfig/power-toggle-config.yaml bin/aws-power-toggle /tmp/build/
cp: can't stat 'bin/aws-power-toggle': No such file or directory
```

**Comment by gbolo** [comment]
@kgeorgiou please fix in branch `release-3.3`

**Comment by kgeorgiou** [comment]
> @kgeorgiou please fix in branch `release-3.3`

@gbolo on it. 

Btw can you give me access to view GitHub Security Alerts? 
We can also turn Dependabot on (from the repo Settings) to automatically update vulnerable dependencies. 🤖

**Comment by gbolo** [comment]
> > @kgeorgiou please fix in branch `release-3.3`
> 
> @gbolo on it.
> 
> Btw can you give me access to view GitHub Security Alerts?
> We can also turn Dependabot on (from the repo Settings) to automatically update vulnerable dependencies.

Thanks @kgeorgiou , sorry to bug you with this issue.
Dependabot seems to be already enabled, but it failed to notice anything lol. As for your access i don't see any option other than "Collaborator"?

**Comment by kgeorgiou** [comment]
Ah no worries, I think GitHub made changes around those features recently. I believe only admins can see the "Dependabot Alerts" section under the "Security" tab.
